src/test/java/api/items/ItemAllowedStatusesSchemaTest.java

```diff
@@ -0,0 +1,103 @@
+package api.items;
+
+import static api.support.InstanceSamples.smallAngryPlanet;
+import static java.nio.file.Files.readAllBytes;
+import static java.nio.file.Paths.get;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.folio.inventory.support.http.client.IndividualResource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import api.support.ApiTests;
+import api.support.builders.HoldingRequestBuilder;
+import api.support.builders.ItemRequestBuilder;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+/**
+ * This test verifies that ramls/item.json and {@link ItemStatusName} is consistent.
+ * <p>
+ * If you're introducing a new status for an item you have to update both item.json
+ * and the {@link ItemStatusName} enum, otherwise item won't pass validation
+ * and will be rejected.
+ */
+@RunWith(JUnitParamsRunner.class)
+public class ItemAllowedStatusesSchemaTest extends ApiTests {
+
+  @Test
+  public void schemaAndEnumIsConsistent() throws IOException {
```

Comment:
Does this demonstrate that the API allows for items to be created with each of the statuses in the interface schema?

Reply:
This tests verifies that item allowed statuses from the `item.json` are the same as in `org.folio.inventory.domain.items.ItemStatusName` enum. We need to keep them sync, otherwise validation will reject item create request, if there is a status in item.json but missed in the enum class.

Reply:
`canCreateItemsWithAllStatuses` in this file demonstrates that behavior.

Reply:
Why do we need both `schemaAndEnumIsConsistent` and `canCreateItemsWithAllStatuses`, it seems like they do nearly the same thing?

Reply:
The `canCreateItemsWithAllStatuses` uses statuses from enum only, but `schemaAndEnumIsConsistent` verifies that the statuses are the same.

Reply:
What do you think to the alternative of creating an item with each status from the schema in one test?

Reply:
Hmm... well the main goal of the test is to prevent a case when someone wanted to add a new status for item but forgot/missed to update either schema or enum.

I don't think if it works.

Reply:
Ok, if the two separate tests mean that one will fail if either of them are missed out, then lets keep both of them